groovy/postInit/chains/Chemistry/VanadiumChain.groovy

```diff
@@ -0,0 +1,78 @@
+import gregtech.api.GregTechAPI;
+import gregtech.api.unification.material.Material;
+
+import static gregtech.api.unification.material.Materials.*;
+import static globals.Globals.*
+
+BR = recipemap('batch_reactor')
+ROASTER = recipemap('roaster')
+MIXER = recipemap('mixer')
+ELECTROLYZER = recipemap('electrolyzer')
+REACTION_FURNACE = recipemap('reaction_furnace')
+EBF = recipemap('electric_blast_furnace')
+
+REACTION_FURNACE.recipeBuilder()
```

Comment:
You cannot immediately separate the solids from one another. Have the recipe create a mixture and leach with water.

Reply:
Also, to separate the chloride from the oxide, treat the mixture with **hot** water, which when cooled will reform lead chloride.